gtsofa/blog
# Active Record  has many  through associations

### What is this PR supposed to do?
- Remove unwanted files .env, .swp etc
- Add a subscriber, magazine and subscription models
- create an association between the above models
### How should this be manually tested?
- clone the repo and navigate to the project directory
- check out this branch
- make migrations 
- start rails console
- create a subscription
### Relevant PT stories
- None
